consistent_web_main/consistent_hash/simple_hash.go

```diff
@@ -0,0 +1,78 @@
+package consistent_hash
```

Comment:
Let's simplify the simplehash logic. We don't need to do multiple replicas for this. I think it is too complicated rn and I am not 100% sure if the logic is correct or not.

The reason why we don't need replicas in simple hash is that for simple hash, we do hash(URL)%N which is by itself uniformly distributed, which is not the case for consistent hashing.
To keep the API unified, you can just send the nodemap to NewSimpleHash as other cases, but not use the number of replicas at all.